test/test_jit.py

```diff
@@ -6614,20 +6614,18 @@ def test_if_list(self):
         # testing that different length lists don't throw error
         @torch.jit.script
         def test_list(x):
-            if True:
+            if bool(x.sum() < 1):
                 c = [x, x]
             else:
                 c = [x, x, x]
             return torch.cat(c)
 
         b = torch.zeros(2, 4)
         test_list.graph.propagate_shapes((b,), False)
-        self.assertExpected(canonical(test_list.graph))
```

Comment:
are we missing a equivalent check for this `test_list`?

Reply:
I wrote this test originally so I know that the intent was just to show that an exception was thrown but that context is missing from looking at the test. Will add something